--- darkplaces/fs.c~	2007-01-21 13:05:32.000000000 +0100
+++ darkplaces/fs.c	2007-01-30 15:28:07.000000000 +0100
@@ -1248,7 +1248,7 @@
 #ifdef DP_FS_BASEDIR
 	strlcpy(fs_basedir, DP_FS_BASEDIR, sizeof(fs_basedir));
 #else
-	strlcpy(fs_basedir, "", sizeof(fs_basedir));
+	strlcpy(fs_basedir, "/usr/share/games/nexuiz", sizeof(fs_basedir));
 
 #ifdef MACOSX
 	// FIXME: is there a better way to find the directory outside the .app?
